src/components/matchHistory/MatchHistory.tsx

```diff
@@ -33,4 +40,33 @@ export const MatchHistory = React.memo(function MatchHistory() {
             />
         </Flex>
     );
-});
\ No newline at end of file
+});
+
+// ADDITIONAL TEST CODE BELOW (AXIOMATICADI)

```

Comment:
let's put this code back into statsReducer before you check in, and remove this code here.

if you really need to call this, you can make the matchesToPlayer an exported function ala "export function matchesToPlayer(..... "  and you can import it in on the MatchHistory.tsx